cmd/datamon/cmd/repo.go

```diff
@@ -28,7 +28,6 @@ var repoDescriptorTemplate *template.Template
 
 func init() {
 	rootCmd.AddCommand(repoCmd)
-	addContextFlag(repoCmd)
```

Comment:
The `addContextFlag`  is also present in `bundle.go`  and `config_set.go`. It should be removed there.
For `context_*.go` we have a special case because the flag is required.

Reply:
Will do.